### PR TITLE
Create chain.json for Maya Protocol

### PR DESCRIPTION
**Pull request by GiMa-Maya**
Addition of Maya Protocol
https://www.mayaprotocol.com
#2005

### CODE REVIEW

File: Maya Protocol/chain.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mayachain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mayaprotocol.com/",
+  "pretty_name": "Maya Protocol",
+  "chain_id": "mayachain-mainnet-v1",
+  "bech32_prefix": "maya",
```
Comment (JeremyParish69) [inline Maya Protocol/chain.json:9]: Please register bech32 prefix here: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: Maya Protocol/chain.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mayachain",
```
Comment (JeremyParish69) [inline Maya Protocol/chain.json:3]: please rename the folder to 'mayachain'

File: Maya Protocol/chain.json [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  "website": "https://mayaprotocol.com/",
+  "pretty_name": "Maya Protocol",
+  "chain_id": "mayachain-mainnet-v1",
+  "bech32_prefix": "maya",
+  "daemon_name": "mayanode",
+  "node_home": "$HOME/.mayanode",
+  "slip44": 931,
+  "codebase": {
+    "git_repo": "https://gitlab.com/mayachain/mayanode",
+    "recommended_version": "mainnet",
```
Comment (JeremyParish69) [inline Maya Protocol/chain.json:15]: it is unheard of to have version names that don't have numbers. This area is not meant to clarify mainnet vs testnet. I'd expect something like 'v1.0'...

**Comment by JeremyParish69** [comment]
Please provide an assetlist.json to accompany

File: Maya Protocol/chain.json (hunk 1 above)
Comment (GiMa-Maya) [reply Maya Protocol/chain.json]: Just created a PR https://github.com/satoshilabs/slips/pull/1542

File: Maya Protocol/chain.json (hunk 3 above)
Comment (GiMa-Maya) [reply Maya Protocol/chain.json]: Updated everything.

File: mayachain/chain.json [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      },
+      {
+        "id": "c0f2a2a7e50b56fc92f5723dd3baf883c12048db",
+        "address": "3.132.55.140:27146"
+      }
+
+    ]
+  },
+  "apis": {
+    "rpc": ["https://tendermint.mayachain.info"],
```
Comment (JeremyParish69) [inline mayachain/chain.json:62]: contained within this array (and the 'rest' array) should be 'endpoint' objects, with an address and provider

File: mayachain/chain.json [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  "chain_id": "mayachain-mainnet-v1",
+  "bech32_prefix": "maya",
+  "daemon_name": "mayanode",
+  "node_home": "$HOME/.mayanode",
+  "slip44": 931,
+  "codebase": {
+    "git_repo": "https://gitlab.com/mayachain/mayanode",
+    "recommended_version": "V1.103.3",
+    "compatible_versions": [
+      " mainnet "
```
Comment (JeremyParish69) [inline mayachain/chain.json:17]: please correct this compatible version to 'V1.103.3'

File: mayachain/chain.json [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    ],
+    "genesis": {
+      "genesis_url": " https://tendermint.mayachain.info/genesis "
+    },
+    "versions": [
+      {
+        "name": " mainnet ",
+        "recommended_version": " mainnet ",
```
Comment (JeremyParish69) [inline mayachain/chain.json:25]: this info (recommended and compatible) should be the same as above in this case.

File: mayachain/chain.json [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+    "recommended_version": "V1.103.3",
+    "compatible_versions": [
+      " mainnet "
+    ],
+    "genesis": {
+      "genesis_url": " https://tendermint.mayachain.info/genesis "
+    },
+    "versions": [
+      {
+        "name": " mainnet ",
```
Comment (JeremyParish69) [inline mayachain/chain.json:24]: this name should encompass all versions that might be compatible with he current version. Perhaps a name like 'V1.103' ?

Comment (GiMa-Maya) [reply mayachain/chain.json:24]: 1.103.3 was crucial. changing it all to 1.103.3

File: mayachain/chain.json [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+  "node_home": "$HOME/.mayanode",
+  "slip44": 931,
+  "codebase": {
+    "git_repo": "https://gitlab.com/mayachain/mayanode",
+    "recommended_version": "V1.103.3",
+    "compatible_versions": [
+      "V1.103.3"
+    ],
+    "genesis": {
+      "genesis_url": " https://tendermint.mayachain.info/genesis "
```
Comment (JeremyParish69) [inline mayachain/chain.json:20]: can we remove the leading and trailing spaces here, please?

File: mayachain/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+          "exponent": 4
+        }
+      ],
+      "base": "maya",
+      "name": "Maya",
+      "display": "MAYA",
+      "symbol": "MAYA",
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mayachain/images/maya.svg",
```
Comment (JeremyParish69) [inline mayachain/assetlist.json:51]: check failed because of the extra comma here

**Pull request merged**